docs/user-guide/python.md

```diff
@@ -128,27 +142,20 @@ variety of scenarios of using Python on the ARCHER2 compute nodes.
     # Load the Python module
     module load cray-python
     
-    # If using a virtual environment
-    source <<path to virtual environment>>/bin/activate
-    
     # Run your Python progamme
     python python_test.py
     ```
 
 !!! tip
-    If you have installed your own packages you will need to set `PATH` and
-    `PYTHONPATH` as described above within your job submission script in order
-    to accesss the commands and packages you have installed.
+    If you have installed your own packages you will need to activate your local Python
+    environment within your job submission script as shown in the previous example.
 
 ### Example mpi4py job submission script
 
-Programs that have been parallelised with mpi4py can be run on
-multiple processors on ARCHER2. A sample submission script is given
-below. The primary difference from the Python submission script in the
-previous section is that we must run the program using
-`srun python my_prog.py` instead of `python my_prog,py`. Failing to do so will 
-cause a segmentation fault in your program when it reaches the line
-"`from mpi4py import MPI`".
+Programs that have been parallelised with mpi4py can be run on multiple processors on ARCHER2.
+A sample submission script is given below. Unlike the serial Python submission script above,
+we must run the program using `srun python mpi4py_test.py` instead of merely `python mpi4py_test,py`.
```

Comment:
Should `srun` have the `--hint=nomultithread --distribution=block:block` options that we use in other parallel examples?